app/src/main/java/ardjomand/leonardo/arcmovies/model/MovieDetails.java

```diff
@@ -0,0 +1,255 @@
+package ardjomand.leonardo.arcmovies.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+/**
+ * Created by unity on 04/03/18.
+ */
+
+public class MovieDetails {
+
+
+    @SerializedName("adult")
```

Comment:
Why map fields that are not being shown?

Reply:
Do you mean the following properties: "production_companies", "production_countries" and "spoken_languages"? I didn't add them because I would not be using them, but I can do it later if that is what you meant.